rotkehlchen/exchanges/kraken.py

```diff
@@ -225,6 +239,19 @@ def history_event_from_kraken(
                 f'Failed to read ledger event from kraken {raw_event} due to {msg}',
             )
             return [], False
+
+    for event_set in receive_spend_events.values():
+        if len(event_set) == 2:
```

Comment:
What if len(event_set) is not 2? This means an error, right? I guess we should warn the user / log it.

Reply:
as explained before is not an error

Reply:
I would add a comment that no action is needed for those other events